src/content/developers/docs/consensus-mechanisms/pos/rewards-and-penalties/index.md

```diff
@@ -0,0 +1,100 @@
+
+Note that empty slots (where a block was not proposed) do not count towards the inclusion distance for attestations from other validators. If a validator is assigned to attest to the 10th slot in an epoch, and there is no block proposed for the 11th slot causing the attestation to be included in a block at slot 12, this would still be considered an inclusion distance of 1. This prevents penalizing active validators for another validator being unavailable.
+
+There are also three additional rewards: `source` (reward for voting for the correct source checkpoint), `target` (reward for voting for the correct target checkpoint), `head` (reward for voting for the correct head block). The value of each of these is equal to `base_reward * inclusion_delay * participation_rate`. The same value can be removed from the validator's balance as a penalty for an incorrect or missing `source`, `target` or `head` vote.
+
+When validators are assigned to sync committees they receive rewards for each slot they sign off. When validators in a sync committee fail to sign blocks they are penalized exactly the value of ether they would have received for signing successfully.
+
+### Attestation scenarios
+
+Assuming attestations are timely (included in the very next block, inclusion distance of 1) the following table describes the attestation rewards for four scenarios:
+
+| `source`          | `target`          | `head`            | `base_reward`                            | penalty            | `inclusion_delay`    | Result                                                            |
+| ----------------- | ----------------- | ----------------- | ---------------------------------------- | ------------------ | -------------------- | ----------------------------------------------------------------- |
+| incorrect/missing | incorrect/missing | incorrect/missing | 0                                        | 3 \* `base_reward` | 0                    | -3 \* `base_reward`                                               |
+| correct           | incorrect/missing | incorrect/missing | `base_reward` \* `participation_rate`    | 2 \* `base_reward` | `base_reward` \* 7/8 | `base_reward` _ `participation_rate` - 9/8 _ `base_reward`        |
+| correct           | correct           | incorrect/missing | 2 _ `base_reward` _ `participation_rate` | `base_reward`      | `base_reward` \* 7/8 | (2 _ `base_reward` _ `participation_rate`) - 1/8 \* `base_reward` |
+| correct           | correct           | correct           | 3 _ `base_reward` _ `participation_rate` | 0                  | `base_reward` \* 7/8 | (3 _ `base_reward` _ `participation_rate`) + `base_reward` \* 7/8 |
```

Comment:
<img width="1517" alt="image" src="https://user-images.githubusercontent.com/54227730/188025893-343fb66b-31ee-45e4-9ee9-5acbe57628a0.png">

- Adds header ID to header
- Using `vote_reward` and `inclusion_reward` to make it more clear these are separate rewards, and being explicit about how the result is the `vote_reward` + `inclusion_reward` - `penalty`
- Moves `inclusion_reward` column to the left to consolidate rewards together
- Simplifies the "results" mathematically, and arranges them all with the `base_reward` multiplier at the end

(same as screenshot above)
```suggestion
### Attestation scenarios {#attestation-scenarios}

Assuming attestations are timely (included in the very next block, inclusion distance of 1) the following table describes the attestation rewards for four scenarios:

| `source`          | `target`          | `head`            | `vote_reward`                              | `inclusion_reward`   | `penalty`          | Result (`vote_reward` + `inclusion_reward` - `penalty`) |
| ----------------- | ----------------- | ----------------- | ------------------------------------------ | -------------------- | ------------------ | ------------------------------------------------------- |
| incorrect/missing | incorrect/missing | incorrect/missing | 0                                          | 0                    | 3 \* `base_reward` |                                    - 3 \* `base_reward` |
| correct           | incorrect/missing | incorrect/missing | 1 \* `base_reward` \* `participation_rate` | 7/8 \* `base_reward` | 2 \* `base_reward` |           (`participation_rate` - 9/8) \* `base_reward` |
| correct           | correct           | incorrect/missing | 2 \* `base_reward` \* `participation_rate` | 7/8 \* `base_reward` | 1 \* `base_reward` |      (2 \* `participation_rate` - 1/8) \* `base_reward` |
| correct           | correct           | correct           | 3 \* `base_reward` \* `participation_rate` | 7/8 \* `base_reward` | 0                  |      (3 \* `participation_rate` + 7/8) \* `base_reward` |
```